docs/rules/no-string-refs.md

```diff
@@ -28,14 +28,17 @@ var Hello = React.createClass({
 
 Valid:
 
-```js
+```jsx
 var Hello = React.createClass({
   componentDidMount: function() {
     var component = this.hello;
     // ...do something with component
   },
+  _helloRef(c){
```

Comment:
at the very least, let's make sure this function does not use the harmful convention of pretending a leading underscore is somehow "private" :-) also, there always needs to be a space between `)` and `{`.